nvm.sh

```diff
@@ -2405,8 +2406,17 @@ nvm() {
       nvm_err "uname -a: '$(command uname -a | command awk '{$2=""; print}' | command xargs)'"
       if [ "$(nvm_get_os)" = "darwin" ] && nvm_has sw_vers; then
         nvm_err "OS version: $(sw_vers | command awk '{print $2}' | command xargs)"
-      elif [ -r "/etc/issue" ]; then
-        nvm_err "OS version: $(command head -n 1 /etc/issue | command sed 's/\\.//g')"
+      else
+        if [ -r "/etc/issue" ]; then
```

Comment:
what if /etc/issue doesn't exist?

Reply:
Sorry that I asked a wrong question.

I wanna say if "/etc/issue" exists, but the content may be empty in some cases, so I put it with the next detection in the same level code block, we can test both of the conditions, otherwise, the existing but empty /etc/issue will cause the next condition and code block not be running.

Will this make sense to you?

Thanks!

Reply:
Not sure if I do explain well, I need the conditions below this one also be tested, even if this conditonal is true.